src/mapping_igvc/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
`CMakeLists` looks good as far as I can see

Reply:
Personally, I'm more of the form of nuking most if not all commented template entries.
Cleaner and easier to parse on first glance - but this is personal preference and I won't go on a murder spree for it :P

Reply:
Hmm, I think it's important that we leave at least some commented entries in for the sections that we are using. While I agree that it would be at least a bit cleaner and easier for us to work with, we should keep in mind that we want to keep this as accessible as possible for new members. For now, I think I'm inclined to keep the comment density as is, but I've asked for some feedback in the #software-leads channel on this, so we can at least have a discussion about it.